x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_content.tsx

```diff
@@ -9,32 +9,31 @@ import React, { FC, useContext, useEffect, useMemo, useState } from 'react';
 import { FieldStats, FieldStatsServices } from '@kbn/unified-field-list-plugin/public';
 import moment from 'moment';
 import { isDefined } from '@kbn/ml-is-defined';
+import { type DataView } from '@kbn/data-plugin/common';
 import { getDefaultDatafeedQuery } from '../../jobs/new_job/utils/new_job_utils';
-import { useMlKibana } from '../../contexts/kibana';
 import { JobCreatorContext } from '../../jobs/new_job/pages/components/job_creator_context';
 import { useFieldStatsFlyoutContext } from './use_field_stats_flytout_context';
 
+export interface TimeRangeMs {
```

Comment:
Since this got duplicated from the `transform` plugin, suggest to avoid that and add this type to the `@kbn/ml-date-picker` package.

Reply:
Thanks for the hint! Updated here [`15484a2` (#149879)](https://github.com/elastic/kibana/pull/149879/commits/15484a2d81ddb770cc9ffa3def370b3d46f0a2b7)